packages/integrations/netlify/test/functions/dynamic-route.test.js

```diff
@@ -21,6 +21,13 @@ describe('Dynamic pages', () => {
 
 	it('Dynamic pages are included in the redirects file', async () => {
 		const redir = await fixture.readFile('/_redirects');
-		expect(redir).to.match(/\/products\/\*/);
+		expect(redir).to.match(/\/products\/:id/);
+	});
+
+	it('Prerendered routes are also included using placeholder syntax', async () => {
+		const redir = await fixture.readFile('/_redirects');
+		expect(redir).to.include('/pets/:cat       /pets/:cat/index.html        200');
+		expect(redir).to.include('/pets/:dog       /pets/:dog/index.html        200');
```

Comment:
Will these conflict? I expect that they might and only `/pets/:cat` would be matched.

Since all of these are generating actual files we might need to track _any_ files that the dynamic route has generated, then add redirects for the exact route?

```
expect(redir).to.include('/pets/cat1       /pets/cat1/index.html        200');
expect(redir).to.include('/pets/cat2       /pets/cat2/index.html        200');
expect(redir).to.include('/pets/cat3       /pets/cat3/index.html        200');
expect(redir).to.include('/pets/dog1       /pets/dog1/index.html        200');
expect(redir).to.include('/pets/dog2       /pets/dog2/index.html        200');
expect(redir).to.include('/pets/dog3       /pets/dog3/index.html        200');
```